modules/hadronRtdProvider.js

```diff
@@ -151,6 +135,17 @@ export function addRealTimeData(bidConfig, rtd, rtdConfig) {
  * @param {Object} userConsent
  */
 export function getRealTimeData(bidConfig, onDone, rtdConfig, userConsent) {
+  if (!storage.getDataFromLocalStorage(LS_TAM_KEY)) {
+    const partnerId = rtdConfig.params.partnerId | 0;
+    const hadronIdUrl = rtdConfig.params && rtdConfig.params.hadronIdUrl;
+    const scriptUrl = urlAddParams(
+      paramOrDefault(hadronIdUrl, HADRON_JS_URL, {}),
+      `partner_id=${partnerId}&_it=prebid`
+    );
+    loadExternalScript(scriptUrl, MODULE_TYPE_RTD, SUBMODULE_NAME, () => {
```

Comment:
A bug fixed coming from the original repository. Seems like there were some interface changes in `loadExternalScript`  function which is not reflected in this module and produced `not whitelisted for loading external JavaScript` error.